folly/memory/Malloc.h

```diff
@@ -145,6 +141,10 @@ inline bool getTCMallocNumericProperty(const char* name, size_t* out) noexcept {
     return true;
   }
 #else
+inline bool getTCMallocNumericProperty(const char* name, size_t* out) noexcept {
+  return MallocExtension_Internal_GetNumericProperty(name, strlen(name), out);
```

Comment:
This weak symbol should always be present - folly declares it -, even if it does not link to anything.

Reply:
not the case unfortunately but anyway only called within this context.

Reply:
Is it a problem to declare these weak symbols also on freebsd?